admin/actions.php

```diff
@@ -250,9 +248,9 @@ function edit_schedule_submit() {
 		}
 	}
 
-	if ( isset( $_POST['hmbkp_schedule_recurrence']['hmbkp_type'] ) ) {
+	$schedule_recurrence_type = sanitize_text_field( $_POST['hmbkp_schedule_recurrence']['hmbkp_type'] );
```

Comment:
I'm not sure why we swapped the lines here, the `if` check should happen before sanitization, so as not to produce an error if the value does not exist.